### PR TITLE
Adding nfs-kernel installation

### PR DESCRIPTION
**Pull request by PravinRanjan10**
…ring

**What this PR does / why we need it**:
Adding nfs-kernel installation and description updation with empty string

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/db/drivers/etcd/etcd.go [hunk 1]
```diff
@@ -578,9 +578,7 @@ func (c *Client) UpdateFileShare(ctx *c.Context, fshare *model.FileShareSpec) (*
 	if fshare.Name != "" {
 		result.Name = fshare.Name
 	}
-	if fshare.Description != "" {
-		result.Description = fshare.Description
-	}
+	result.Description = fshare.Description
```
Comment (leonwanghui) [inline pkg/db/drivers/etcd/etcd.go]: It's not allowed to update empty description.

Comment (wisererik) [reply pkg/db/drivers/etcd/etcd.go]: +1

Comment (PravinRanjan10) [reply pkg/db/drivers/etcd/etcd.go]: +1

**Comment by xing-yang** [comment]
Please add a release note.  Also please add explanation why this is needed in the PR description.

**Pull request merged**